src/dom/dom.js

```diff
@@ -3456,7 +3456,7 @@ p5.File._createLoader = function(theFile, callback) {
 p5.File._load = function(f, callback) {
   // Text or data?
   // This should likely be improved
-  if (/^text\//.test(f.type)) {
+  if (/^text\/|json/.test(f.type)) {
```

Comment:
I think you can directly test for `application/json` as there shouldn't be any other MIME types for JSON.

Reply:
Done  :+1: